### PR TITLE
config: allow configuring button and touch bindings

### PR DESCRIPTION
**Pull request by grahamc**

### CODE REVIEW

File: mako.5.scd [hunk 1]
```diff
@@ -59,6 +59,25 @@ Empty lines and lines that begin with # are ignored.
 
 	Default: top-right
 
+# BUTTON AND TOUCH BINDINGS
+
+Specify the action to take per button or touch event. Supported values are
+_none_, _dismiss_, _dismiss-all_, _dismiss-group_, _invoke-default-action_,
+and _bottom-center_.
```
Comment (emersion) [inline mako.5.scd]: Is "bottom-center" a leftover from a copy-pasta?

**Comment by emersion** [comment]
Otherwise looks good!

File: mako.5.scd (hunk 1 above)
Comment (grahamc) [reply mako.5.scd]: d'oh, yep, I even have an uncommitted, staged patch here fixing that.

**Comment by grahamc** [comment]
Fixup'd!

**Pull request merged**